### PR TITLE
Add `>>`, `>>=` operators

### PR DESCRIPTION
**Pull request by Luhrel**
Also add missing `<<` and `<<=` operators

### CODE REVIEW

File: src/gmp/z.d [hunk 1]
```diff
@@ -562,6 +562,7 @@ nothrow:
             else static if (s == "|") __gmpz_ior(mut_rhs._ptr, _ptr, rhs._ptr);
             else static if (s == "^") __gmpz_xor(mut_rhs._ptr, _ptr, rhs._ptr);
             else static if (s == "<<") __gmpz_mul_2exp(mut_rhs._ptr, _ptr, cast(ulong)rhs);
+            else static if (s == ">>") __gmpz_tdiv_q_2exp(mut_rhs._ptr, _ptr, cast(ulong)rhs);
```
Comment (nordlow) [inline src/gmp/z.d]: We need to handle the cases where `rhs` is negative for both `<<` and `>>`, right?

File: src/gmp/z.d [hunk 2]
```diff
@@ -588,6 +589,7 @@ nothrow:
             else static if (s == "|") __gmpz_ior(y._ptr, _ptr, rhs._ptr);
             else static if (s == "^") __gmpz_xor(y._ptr, _ptr, rhs._ptr);
             else static if (s == "<<") __gmpz_mul_2exp(y._ptr, _ptr, cast(ulong)rhs);
+            else static if (s == ">>") __gmpz_tdiv_q_2exp(y._ptr, _ptr, cast(ulong)rhs);
```
Comment (nordlow) [inline src/gmp/z.d]: We need to handle the cases where `rhs` is negative for both `<<` and `>>`, right?

Comment (Luhrel) [reply src/gmp/z.d]: Mmmh right.

If `rhs` is negative, shall I disallow it (`assert(0);`), or simply reconverting it to the opposite shift (`z >> -1` becomes `z << 1` ?

Comment (nordlow) [reply src/gmp/z.d]: I vote for the latter. I prefer avoiding error states.

**Comment by nordlow** [comment]
Awesome! Thanks a lot.

**Comment by nordlow** [comment]
Pushed tag v0.2.13.

**Pull request merged**